### ISSUE TITLE
WebIDL syntax issues

### ISSUE THREAD

**Issue by foolip** [w3c/quota-api#5]
http://w3c.github.io/quota-api/#storagequota-interface

``` WebIDL
[NoInterfaceObject]
interface StorageQuota {
    [SameObject]
    readonly        attribute frozen array supportedTypes;
    Promise queryInfo (StorageType type);
    Promise requestPersistentQuota ([Clamp] unsigned long long newQuota);
};
```

This should use `FrozenArray<StorageType>` and `Promise<void>`.

**Comment by foolip** [comment]
Ping?

**Comment by marcoscaceres** [comment]
I fixed this as part #7

**Comment by marcoscaceres** [comment]
Whoops, didn't mean to close.
